src/index.ts

```diff
@@ -126,7 +127,7 @@ export default function create<TState extends State>(
     const forceUpdate = useReducer(forceUpdateReducer, 1)[1]
     useIsoLayoutEffect(() => subscribe(forceUpdate, options), [])
 
-    return options.currentSlice as StateSlice
+    return options.currentSlice as ReturnState<TState, StateSlice>
```

Comment:
The `as {TYPE}` can be removed because all possible return types would be covered in the interface.

Reply:
Done